### PR TITLE
MV backpressure yaml changes

### PR DESCRIPTION
**Pull request by juliayakovlev**
## PR pre-checks (self review)

- [ ] ~~I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)~~
- [ ] ~~I gave variables/functions meaningful self-explanatory names~~
- [ ] ~~I didn't leave commented-out/debugging code~~
- [ ] ~~I didn't copy-paste code~~
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [ ] ~~All new and existing unit tests passed (`hydra unit-tests`)~~
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

Changes:
1. Rename test yaml (it's feature, not longevity)
2. Rename Jenkins pipeline (it's feature, not longevity)
3. Disable HH for the Scylla (not need for backpressure test)
4. Add "nemesis_interval: 30" to the test_defaults.yaml

### CODE REVIEW

**Comment by roydahan** [comment]
According to @eliransin the current test in master doesn't exercise the MV backpressure.
Now, there are metrics that can follow this mechanism better (@eliransin please provide name).

@juliayakovlev, please change the test to:
1. Have more MVs so we can exercise the backpressure.
2. Validate the mechanism works according the metrics and assert according to the metrics if it doesn't behave correctly (first need to try some runs in order to define it).
3. Reduce the time of the test significantly (one to several hours).

**Comment by bentsi** [comment]
@juliayakovlev please address comments

**Comment by fruch** [comment]
@juliayakovlev please resolve conflicts
@roydahan please approve the PR, and that we want to backport it to 3.1

**Comment by juliayakovlev** [comment]
> @juliayakovlev please resolve conflicts
> @roydahan please approve the PR, and that we want to backport it to 3.1

@fruch 
This feature is not ready. We are waiting for developers

**Comment by roydahan** [comment]
@juliayakovlev no need to wait for dev.
We can merge this and test later with them.
Just make sure all changes are still relevant, for example I think that the json with dashboard isn't needed, is it?

File: defaults/test_default.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ space_node_threshold: 0
 reuse_cluster: false
 nemesis_class_name: 'NoOpMonkey'
 nemesis_during_prepare: 'true'
+nemesis_interval: 30
```
[the remark this answers is not in the document]
Comment (bentsi) [reply defaults/test_default.yaml]: there is one already set

[the remark this answers is not in the document]
Comment (juliayakovlev) [reply defaults/test_default.yaml]: I see that nemesis_interval is added by you  with 5 min interval in the October.
I put this a lot time before

File: test-cases/features/2mv-backpressure-4d.yaml [hunk 2]
```diff
@@ -11,3 +11,7 @@ user_prefix: 'longevity-2mv-backpressure-4d'
 
 instance_type_db: 'i3.2xlarge'
 instance_type_loader: 'c5.2xlarge'
+
+nemesis_during_prepare: 'false'
```
Comment (bentsi) [inline test-cases/features/2mv-backpressure-4d.yaml]: these are booleans, why putting ''?

**Comment by bentsi** [comment]
@juliayakovlev per Roy's question, do we need the JSON changes?

File: test-cases/features/2mv-backpressure-4d.yaml (hunk 2 above)
Comment (juliayakovlev) [reply test-cases/features/2mv-backpressure-4d.yaml]: Fixed

**Comment by juliayakovlev** [comment]
> @juliayakovlev no need to wait for dev.
> We can merge this and test later with them.
> Just make sure all changes are still relevant, for example I think that the json with dashboard isn't needed, is it?

@roydahan  @bentsi  I think we need this graph, because we can validate the MV backpressure using this graph

**Pull request merged**